src/scss/_sofie-color-vars.scss

```diff
@@ -8,7 +8,9 @@ body {
   --black-color: #000000;
   --dark-color: #3a3a3a;
   --background-color: #343434;
+  --red-color: red;
   --green-color: green;
+  --bright-green-color: #00AC17;
```

Comment:
`--read-color` and `--bright-green-color` seems to be used for the OnAir- and NextBorder colors of the MiniShelves.
We already have the variables `--on-air-color` and `--next-color` on line 19 and 20 in this file. These variables should be used, so we don't have multiple variables for the same thing (thus defeating the point of the variables)